v03_pipeline/bin/migration.py

```diff
@@ -0,0 +1,587 @@
+import hail as hl
+
+from v03_pipeline.lib.misc.family_entries import globalize_ids
+from v03_pipeline.lib.misc.io import import_pedigree, write
+from v03_pipeline.lib.misc.pedigree import parse_pedigree_ht_to_families
+from v03_pipeline.lib.model import DatasetType, Env, ReferenceGenome
+
+# Need to use the GCP bucket as temp storage for very large callset joins
+hl.init(tmp_dir='gs://seqr-scratch-temp', idempotent=True)
+
+# Interval ref data join causes shuffle death, this prevents it
+hl._set_flags(use_new_shuffle='1', no_whole_stage_codegen='1')  # noqa: SLF001
+sc = hl.spark_context()
+sc.addPyFile('gs://seqr-luigi/releases/dev/latest/pyscripts.zip')
+
+
+def deglobalize_sample_ids(ht: hl.Table) -> hl.Table:
+    ht = ht.annotate(
+        entries=(
+            hl.enumerate(ht.entries).starmap(
+                lambda i, e: hl.Struct(**e, s=ht.sample_ids[i]),
+            )
+        ),
+    )
+    return ht.drop('sample_ids')
+
+
+Env.HAIL_TMPDIR = 'gs://seqr-scratch-temp'
+
+MIGRATIONS = [
+    (
+        DatasetType.SNV_INDEL,
+        ReferenceGenome.GRCh37,
+        [
+            'gs://seqr-hail-search-data/v03/GRCh37/SNV_INDEL/projects/R0200_bonnemann_int92_ext123s.ht',
```

Comment:
I feel like it would be more precise if we di a `gsutil ls` and then ran this on all the projects that are there, instead of hardcoding a list?

Reply:
Yeah, that's reasonable.  I was just.... incredibly lazy here.